mozilla/blurts-server
# Adds email preview for email verification and breach notification

# References: 
Jira:
- [MNTOR-1056](https://mozilla-hub.atlassian.net/browse/MNTOR-1056)
- [MNTOR-1107](https://mozilla-hub.atlassian.net/browse/MNTOR-1107)



# Description

This PR migrates the breach alert email and adds a preview page from where we can also send test emails.

# Screenshot

<img width="937" alt="image" src="https://user-images.githubusercontent.com/13835474/217681753-e9bbb332-de68-4e32-9f79-c2e120c31a4d.png">

# How to test

1. Set the `.env` variable `ADMINS=` so you can navigate to `/admin/emails`
2. Set `EMAIL_TEST_RECIPIENT=localmonitor20200827@mailinator.com` — this is the same public Mailinator inbox address we already use. For more info see [README](https://github.com/mozilla/blurts-server#trigger-breach-alert-email).
3. You can toggle between email preview templates using the select input and send test emails to your primary or the Mailinator email address mentioned in the previous step.

# Checklist (Definition of Done)
